llvm/include/llvm/ProfileData/Coverage/CoverageMapping.h

```diff
@@ -557,6 +558,35 @@ struct MCDCRecord {
   }
 };
 
+/// Compute Conds[].Idx from Branch-like structure
+class MCDCTVIdxBuilder {
+public:
+  struct MCDCNode {
+    int InCount = 0; /// Reference count; temporary use
+    int Width;       /// Number of paths (>= 1)
```

Comment:
Why not name it `NumPaths`?

Reply:
It was `W` in my prototype design. I have to update my imagination if this would be renamed.
Or, does the comment mislead?

Reply:
https://en.wikipedia.org/wiki/Width_(disambiguation) Width has many meanings for graphs. :)

Reply:
Which is better for me to rename to easy `N` or to define `Width` here.
I haven't learnt the graph theory though, looks like I have introduced similar concept, as far as I read introductions of the theory.

Let me think more.